code/modules/organs/prosthetics/prosthetics_manufacturer.dm

```diff
@@ -5,7 +5,7 @@
 	limb_tech = "{'engineering':1,'materials':1,'magnets':1}"
 	modifier_string = "robotic"
 	is_robotic = TRUE
-	body_flags = BODY_FLAG_NO_DNA | BODY_FLAG_NO_PAIN | BODY_FLAG_NO_EAT
+	body_flags = BODY_FLAG_NO_DNA | BODY_FLAG_NO_DEFIB | BODY_FLAG_NO_STASIS | BODY_FLAG_NO_PAIN | BODY_FLAG_NO_EAT
 	material = /decl/material/solid/metal/steel
 	/// Determines which bodyparts can use this limb.
```

Comment:
Prosthetic eyes had this set in eyes.dm, it should probably be moved to here.
```dm
		flash_mod = 1
		darksight_range = 2
```
Given that prosthetic eyes would overwrite species darksight (and maybe rightfully so?), I feel like maybe we should include this still as some sort of compensation.